src/eventDispatcher/event/Event.ts

```diff
@@ -1,24 +1,34 @@
-
 /**
- * Basic event class which holds event type and data properties and which should be extended
- * by any custom event class.
+ * Basic event class which holds event type and data properties and which can be used as it is or
+ * could be extended by any custom event class.
  * @author Jānis Radiņš
  */
 export class Event {
 
+    private _defaultPrevented:boolean = false;
+
     /**
-     * Event string type
+     * Create new instance
+     * @param {string} type Event string type
+     * @param data Data shipped along with event notification, if any
      */
-    readonly type:string;
+    constructor(public readonly type:string,
+                public readonly data?:any
+    ) {}
 
     /**
-     * Data shipped along with event notification, if any
+     * Flag which indicates that somewhere within event listeners default event action has been prevented
+     * @returns {boolean}
      */
-    readonly data:any;
+    get defaultPrevented(): boolean {
```

Comment:
There shouldn't be space between the boolean type definition.
A linter should be set up eventually.

Reply:
That's how IDE is formatting new getters

Reply:
Each IDE may have different settings, but we should keep the style that is used in the repo.

Reply:
A linter must be added at a later point and this should be corrected.